dxr/static_unhashed/css/icons.css

```diff
@@ -137,6 +137,9 @@ a.type {
 .html {
     background-image: url('/static/icons/mimetypes/html.png');
 }
+.icon-container.folder, .promoted-icon.folder {
+    background: transparent url('/static/icons/mimetypes/folder.png');
```

Comment:
Nit: I think the rest of these are alphabetized by the image name/class name, `folder` in this case.